### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes several typos I spotted on the website.

### CODE REVIEW

**Review by Murderlon — APPROVED** [review]
Very nice, thank you

**Comment by deining** [comment]
> Very nice, thank you

You are welcome.

I attempted to get the checker green via a forced push, which disabled `auto-merge`. I can't see the check results right now. Can you re-approve, plase? Thanks.

**Review by Murderlon — APPROVED** [review]

**Comment by deining** [comment]
@Murderlon: I ran the prettier and now there are no errors reported any more for me on my local machine. Can you please re-appove? (hopeffuly for the last time!). Thanks.

File: docs/uploader/tus.mdx [hunk 1]
```diff
@@ -279,7 +279,7 @@ server you use how it’s done exactly. [`tusd`][], for instance, exposes
 ### Which server do you recommend?
 
 [Transloadit](https://transloadit.com) runs [`tusd`][] in production, where it
-serves millions of requests globally. Therefor we recommend `tusd` as
+serves millions of requests globally. Therefore we recommend `tusd` as
```
Comment (Murderlon) [inline docs/uploader/tus.mdx:282]:
```suggestion
serves millions of requests globally. So we recommend `tusd` as
```

**Pull request merged**